src/VCEL.Core/Expression/JSParse/ToJSFunction.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using VCEL.Monad;
+
+namespace VCEL.Core.Expression.JSParse
+{
+    internal class ToJSFunction : IExpression<string>
+    {
+        private static Dictionary<string, string> JSFunctionMap
+             = new Dictionary<string, string>()
+             {
+                 { "abs", "Math.abs" },
+                 { "round", "Math.round" },
+                 { "sqrt", "Math.round" },
+                 { "min", "Math.min" },
+                 { "max", "Math.max" },
+                 { "now", "Date.now" },
+                 { "today", "new Date" },
```

Comment:
`Date.now()` returns a number rather than a Date object.
`new Date()` might be a better fit.
The today JS implementation is going to be more complex than just `new Date`, perhaps worth us solving as another task?